src/core/crypto_key_mgmt.c

```diff
@@ -474,16 +514,14 @@ int32_t Crypto_Key_verify(TC_t *tc_frame)
         // Initialization Vector
         for (y = 0; y < SDLS_IV_LEN; y++)
         {
-            sdls_frame.pdu.data[pdu_data_idx] = 0x00;
-            sdls_ep_keyv_reply.blk[x].iv[y]   = 0x00;
-            sdls_ep_reply[pdu_data_idx]       = 0x00;
+            sdls_ep_keyv_reply.blk[x].iv[y] = 0x00; //= *(tc_frame->tc_sec_header.iv + y);
+            sdls_ep_reply[pdu_data_idx]     = 0x00; //= *(tc_frame->tc_sec_header.iv + y);
             pdu_data_idx += 1;
             count += 1;
         }
         // ***** This increments the lowest bytes of the IVs so they aren't identical
-        sdls_frame.pdu.data[pdu_data_idx - 1] = sdls_frame.pdu.data[pdu_data_idx - 1] + x + 1;
-        sdls_ep_keyv_reply.blk[x].iv[11]      = sdls_ep_keyv_reply.blk[x].iv[SDLS_IV_LEN - 1] + x + 1;
-        sdls_ep_reply[pdu_data_idx - 1]       = sdls_ep_reply[pdu_data_idx - 1] + x + 1;
+        sdls_ep_keyv_reply.blk[x].iv[SDLS_IV_LEN - 1] = sdls_ep_keyv_reply.blk[x].iv[SDLS_IV_LEN - 1] + x + 1;
+        sdls_ep_reply[pdu_data_idx - 1]               = sdls_ep_reply[pdu_data_idx - 1] + x + 1;
```

Comment:
Any concerns about overflow? Should we use the special increment function for IV here?

Reply:
@jlucas9 I don't think there's a huge issue above, but welcome feedback.